Kudu.Core/Environment.cs

```diff
@@ -270,6 +271,22 @@ public string SiteExtensionSettingsPath
             get { return _siteExtensionSettingsPath; }
         }
 
+        public string FunctionsPath
+        {
+            get
+            {
+                return this.WebRootPath;
+            }
+        }
+
+        public string AppBaseUrlPrefix
+        {
+            get
+            {
+                return HttpContext.Current?.Request?.Url?.GetLeftPart(UriPartial.Authority);
```

Comment:
I did it like that to mirror how the [webjobs manager does it](https://github.com/projectkudu/kudu/blob/master/Kudu.Core/Jobs/JobsManagerBase.cs#L458-L470). We shouldn't be throwing a null ref though, right? I can throw InvalidOperation I think.